pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -35,6 +36,23 @@ def test_min_max(self):
         assert _min == "d"
         assert _max == "a"
 
+    @pytest.mark.parametrize(
+        "categories,expected",
+        [
+            (list("ABC"), np.NaN),
+            ([1, 2, 3], np.NaN),
+            (Series(date_range("2020-01-01", periods=3), dtype="category"), np.NaN),
```

Comment:
Is the `np.NaN` expected? I would perhaps expect `pd.NaT` here.

Note that `CategoricalDtype.na_value` looks buggy for other dtypes, since it's hard coded to `np.nan`.

Reply:
You're absolutely right, I had `pd.NaT` there before and it made the tests fail (Both with `na_value_for_dtype` and `self.dtype.na_value`). I thought @jreback 's reply here (https://github.com/pandas-dev/pandas/pull/30227#discussion_r357653869) meant that I should put `NaN` in the test for now, as this might be related to https://github.com/pandas-dev/pandas/issues/29962.

Maybe I can put `pd.NaT` back in the test and mark it as xfail for now?

Reply:
yes let's xfail *only* that particular test for now, ping on green.